sdk/storage/storage-queue/review/storage-queue.api.md

```diff
@@ -259,14 +260,6 @@ export interface MessagesDequeueHeaders {
     version?: string;
 }
 
-// @public
-export interface MessagesDequeueOptionalParams extends CommonOptions {
```

Comment:
I don't think these `*OptionalParams` are used anywhere in our public surface, but I can put them back if there are any concerns.

Reply:
I would keep them all on the public surface area just in case of a break.

Reply:
they are back now.